features/step_definitions/basic_steps.rb

```diff
@@ -31,6 +31,14 @@ def page_path_from(page_name)
   end
 end
 
-Given("I provide invalid credentials") do
+Given('I provide invalid credentials') do
   OmniAuth.config.mock_auth[:spotify] = :invalid_credentials
 end
+
+Then('I should be on {string} page') do |_string|
```

Comment:
Done

Reply:
@OskarCA What do you mean here when you say `done`? 🤔 

I assume the reason why you're using `_link` is to silence the linter because you've declared a variable that you're not using in the current block. If you're going to have individual steps for visiting each page in your app then you should just define them without extracting the page name into a string

instead of `I should be on "Ticket sales" page` for example have `I should be on the Ticket sales page`

Have a look at [this comment](https://github.com/CraftAcademy/evensteven/pull/10#discussion_r171255271) for a suggestion on how you could have a single step that takes care of every pages